tox.ini

```
@@ -15,6 +15,8 @@ extras =
     coverage: coverage
 deps =
     pipsupported: pip==24.2
+    pipsupported: setuptools<=75.8.2
```

nit: let's make new things sparse.
```suggestion
    pipsupported: setuptools <= 75.8.2
```